performance/performance_tuning.rst

```diff
@@ -49,12 +103,67 @@ Lastly, you can create indexes on your tables to enhance database performance. I
 
 For write performance, you can use general PostgreSQL configuration tuning to increase INSERT rates. We commonly recommend increasing checkpoint_timeout and max_wal_size settings. Also, depending on the reliability requirements of your application, you can choose to change fsync or synchronous_commit values.
 
+Once you have tuned a worker to your satisfaction you will have to manually apply those changes to the other workers as well. To verify that they are all behaving properly, set this configuration variable on the master:
+
+::
+
+  SET citus.explain_all_tasks = 1;
```

Comment:
Minor suggestion.

I was going over https://github.com/citusdata/citus/issues/621 and ended up here.

Is it worth highlighting here that enabling this flag notably hurts ``EXPLAIN ANALYZE`` performance? I'm making this suggestion for two reasons:

1. When the user runs ``EXPLAIN ANALYZE``, they are likely investigating a performance problem. By enabling this flag, they are likely making the performance issue much worse.
2. Citus' ``EXPLAIN ANALYZE`` behavior is inconsistent with that of Postgres'.